docs/integrations/builtin/credentials/google/oauth-generic.md

```diff
@@ -100,3 +104,6 @@ n8n doesn't support all scopes. When creating a generic Google OAuth2 API creden
 
 --8<-- "_snippets/integrations/builtin/credentials/google/unverified-app.md"
 
+### Google Cloud app becoming unauthorized
```

Comment:
For this section on both pages, it looks like the content is identical? So let's turn it into a snippet. Snippets let us write once and reuse everywhere, which makes it easier to update in future.

To add a snippet:

1. Create a file in `_snippets`, at the correct location (the `_snippets` structure largely mirrors the site structure, so in this case, create it in the google creds snippets directory)
2. Add the snippet where you want to use it, like this: `--8<-- "_snippets/integrations/builtin/credentials/google/enable-apis.md"`

In this case, please create a snippet for "Google Cloud app becoming unauthorized", and add it on both pages, in the same way as I did for "Google hasn't verified this app"

The reason we don't include the section heading in the snippet: it makes it easier for writers, as we can still see the structure of the page, and know at a glance roughly what the snippet is about.